src/Templates/cs/Templates/CustomTypeEditorTemplatePack.csproj

```diff
@@ -10,17 +10,26 @@
     <PackageTags>dotnet-new;templates;WinForms;TypeEditor</PackageTags>
 
     <TargetFramework>netstandard2.0</TargetFramework>
-
+    <LangVersion>10.0</LangVersion>
     <IncludeContentInPack>true</IncludeContentInPack>
     <IncludeBuildOutput>false</IncludeBuildOutput>
     <ContentTargetFolders>content</ContentTargetFolders>
     <NoWarn>$(NoWarn);NU5128</NoWarn>
```

Comment:
Might be useful to add an explanation for the exclusion

Reply:
I think about it.